LidarMap.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+class LidarMap:
+    def __init__(self, lidar_scan, size=11, resolution=1):
+        """
+        Initialize the map using a Lidar scan.
+        :param lidar_scan: An object with properties of the Lidar scan.
+        :param size: The width and height of the square map in cells.
+        :param resolution: The size of each cell in meters.
+        """
+        self.size = size  # size of the map in cells
+        self.resolution = resolution  # resolution of a cell in meters
+        self.map = np.zeros((size, size), dtype=bool)  # initialize the map with boolean values
+        self.position = (size // 2, size-1)  # point that the lidar is taking scan from
+        self.process_scan(lidar_scan)
+
+    def process_scan(self, lidar_scan):
+        """
+        Process the Lidar scan and update the map accordingly.
+        :param lidar_scan: An object with properties of the Lidar scan.
+        """
+        angle = lidar_scan.angle_min
+
+        for index, distance in enumerate(lidar_scan.ranges):
```

Comment:
Możemy użyć `distance, angle in zip(lidar_scan.ranges, range(lidar_scan.angle_min, lidar_scan.angle_max)`

Reply:
Ten range to lepiej nawet stworzyć wyżej i nazwać go angles